src/controllers/expensesController.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+
+const expensesService = require('../services/expensesServices');
+const usersService = require('../services/usersServices');
+
+beforeEach(() => {
+  expensesService.deleteAllExpenses();
+});
+
+const getExpenses = (req, res) => {
+  const {
+    userId,
+    from,
+    to,
+    categories,
+  } = req.query;
+
+  let expenses = expensesService.getExpenses();
+
+  if (userId) {
+    expenses = expenses.filter(expense => expense.userId === +userId);
```

Comment:
```suggestion
    expenses = expenses.filter(expense => expense.userId === Number(userId));
```

The same in all other places, it's easier for understanding